android_webview/test/shell/tango/jni/TangoHandler.cpp

```diff
@@ -746,6 +748,49 @@ bool TangoHandler::getPlanes(std::vector<Plane>& planes) {
   return false;
 }
 
+bool TangoHandler::getPlaneDeltas(PlaneDeltas& planeDeltas) {
+  if (!connected)
+  {
+    return false;
+  }
+
+  std::vector<Plane> planes;
+  getPlanes(planes);
+
+  // Find all the planes that have been added or changed this frame.
+  std::unordered_map<long, Plane> planeMapNew;
+  for (auto plane : planes) {
+    planeMapNew[plane.identifier] = plane;
+    // Check if this is a plane we already know about.
+    auto prev = planeMap.find(plane.identifier);
+    if (prev != planeMap.end()) {
+      // This is an existing plane. Has it changed?
+      Plane existing = prev->second;
+      if (existing.timestamp != plane.timestamp) {
+        // The plane has a new timestamp, thus it has changed.
+        planeDeltas.updated.push_back(plane);
+        LOGE("PLANE UPDATED!@!!!!!!!!!!!!");
+      }
+    } else {
+      // This is a new plane.
+      planeDeltas.added.push_back(plane);
+      LOGE("PLANE ADDED!@!!!!!!!!!!!!");
```

Comment:
I would encourage to get rid of these error logs. LOGE is for errors and LOGI for information messages.

Reply:
done